boolean/boolean.py

```diff
@@ -363,10 +363,16 @@ class Symbol(Expression):
 
     _obj = None
 
-    def __new__(cls, obj=None, eval=False):
+    def __new__(cls, obj=None, eval=True):
         return object.__new__(cls)
 
-    def __init__(self, obj=None, eval=False):
+    def __init__(self, obj=None, *args, **kwargs):
```

Comment:
why do you need variadic args at all here? I reckon this is for consistency, but may be we do not need them at all on any of the initializers?